terraform/modules/k8s_node/vms.tf

```diff
@@ -104,7 +104,7 @@ resource "null_resource" "cluster" {
   count      = var.k8s_cluster.sandbox_status ? 0 : 1
   depends_on = [local_file.AnsibleInventory, local_file.AnsibleConfig, vsphere_virtual_machine.vm]
   provisioner "local-exec" {
-    command = "ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -b -i ${var.ansible_dir}/inventory/nodes.ini ${var.ansible_dir}/cluster.yml"
+    command = "ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -b -i ${var.ansible_dir}/inventory/nodes.ini ${var.ansible_dir}/cluster.yaml"
```

Comment:
This file is actually cluster.yml. Check in the ansible folder.

Reply:
Let me re-name the file!